pravega-cli/src/main/resources/config.properties

```diff
@@ -10,20 +10,24 @@
 
 # Config for CLI credentials (at the moment, Controller auth is only supported).
 cli.authEnabled=false
+cli.tlsEnabled=false
 cli.userName=admin
 cli.password=1111_aaaa
 
+# Config for CLI security
+cli.security.tls.trustStore.location=./conf/client.truststore.jks
+
 # Config for CLI Controller REST/GRPC endpoints.
-cli.controllerRestUri=http://localhost:10080
-cli.controllerGrpcUri=tcp://localhost:9090
+cli.controllerRestUri=localhost:9091
+cli.controllerGrpcUri=localhost:9090
 cli.metadataBackend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
 pravegaservice.clusterName=pravega
 pravegaservice.containerCount=4
 
 # Config for Zookeeper service.
-pravegaservice.zkURL=localhost:2181
+pravegaservice.zkURL=localhost:4000
```

Comment:
Why this has changed?

Reply:
By default, the standalone's zk server runs on the port 4000